pennylane/ops/op_math/controlled_decompositions.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2018-2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This submodule defines functions to decompose controlled operations
+"""
+
+import pennylane as qml
+from pennylane.operation import Operator
+from pennylane.wires import Wires
+
+
+def ctrl_decomp_zyz(target_operation: Operator, control_wires: Wires):
+    """Decompose a controlled single-qubit operation given the target operation
+
+    This function decomposes the controlled version of a single-qubit operation
+    given a target operation using the decomposition defined in section 5 of
+    `Barenco et al. (1995) <https://arxiv.org/abs/quant-ph/9503016>`_.
```

Comment:
Do we need to say it twice? Just a suggestion:

```suggestion
    """This function decomposes the controlled version of a single-qubit operation
    given a target operation using the decomposition defined in section 5 of
    `Barenco et al. (1995) <https://arxiv.org/abs/quant-ph/9503016>`_.
```

Reply:
I think the first line is important for the docs so that there is a one line description that can be rendered.